addons/project_issue/models/project_issue.py

```diff
@@ -0,0 +1,335 @@
+
+            dt_create_date = fields.Datetime.from_string(issue.create_date)
+            if issue.date_open:
+                dt_date_open = fields.Datetime.from_string(issue.date_open)
+                issue.day_open = (dt_date_open - dt_create_date).total_seconds() / (24.0 * 3600)
+                issue.working_hours_open = Calendar._model.get_working_hours(self.env.cr, self.env.uid, calendar_id, dt_create_date, dt_date_open,
+                                                                             compute_leaves=True, resource_id=False, default_interval=(8, 16),
+                                                                             context=self.env.context)
+
+            if issue.date_closed:
+                dt_date_closed = fields.Datetime.from_string(issue.date_closed)
+                issue.day_close = (dt_date_closed - dt_create_date).total_seconds() / (24.0 * 3600)
+                issue.working_hours_close = Calendar._model.get_working_hours(self.env.cr, self.env.uid, calendar_id, dt_create_date, dt_date_closed,
+                                                                              compute_leaves=True, resource_id=False, default_interval=(8, 16),
+                                                                              context=self.env.context)
+
+    @api.depends('create_date', 'date_action_last', 'date_last_stage_update')
+    def _compute_inactivity_days(self):
+        current_datetime = fields.Datetime.from_string(fields.Datetime.now())
+        for issue in self:
+            dt_create_date = fields.Datetime.from_string(issue.create_date)
+            issue.days_since_creation = (current_datetime - dt_create_date).days
+
+            if issue.date_action_last:
+                issue.inactivity_days = (current_datetime - fields.Datetime.from_string(issue.date_action_last)).days
+            elif issue.date_last_stage_update:
+                issue.inactivity_days = (current_datetime - fields.Datetime.from_string(issue.date_last_stage_update)).days
+            else:
+                issue.inactivity_days = (current_datetime - dt_create_date).days
+
+    @api.onchange('partner_id')
+    def onchange_partner_id(self):
+        """ This function sets partner email address based on partner
+        """
+        self.email_from = self.partner_id.email
+
+    @api.multi
+    def on_change_project(self, project_id):
+        if project_id:
+            project = self.env['project.project'].browse(project_id)
+            return {'value': {'partner_id': project.partner_id.id}}
+
+    @api.onchange('task_id')
+    def onchange_task_id(self):
+        self.user_id = self.task_id.user_id
+
+    @api.multi
+    def copy_data(self, default=None):
+        self.ensure_one()
+        if not default:
+            default = {}
+        default.update(name=_('%s (copy)') % (self.name or ""))
+        return super(ProjectIssue, self).copy_data(default)[0]
```

Comment:
well @hmo-odoo copy_data of super will return list(as it is decorated with cr_uid_id_context using guess mechanism), I would rather prefer to override copy method(if there is no need of overriding copy_data)